sn_networking/src/lib.rs

```diff
@@ -1017,6 +1017,11 @@ impl Network {
                             NetworkAddress::from_record_key(&record_key),
                         ));
                     }
+                    Err(NetworkError::GetRecordError(GetRecordError::SplitRecord { .. }))
```

Comment:
`non Wasm` build has a handling code `` trying to merge `Registers`, i.e. here get Register won't emit a SplitRecord.

not sure about Vault, but if it is a crdt type, then a merge shall be undertaken as well.

meanwhile, the `Wasm` build donesn't try to carry out `merge` yet, which could be an issue.

Swallowing an error here is also bit concerning, hence I'd suggest give it a proper handling here (if emits) or still keeps raise the error up.